### PR TITLE
Adjust to yiisoft/di changes

### PR DESCRIPTION
**Pull request by razonyang**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  |

### CODE REVIEW

File: config/common.php [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <?php
 
-use yii\di\Reference;
+use Yiisoft\Factory\{Factory, Definitions\Reference};
```
Comment (hiqsol) [inline config/common.php]: @samdark Are we ok with this syntax? :)

Comment (viktorprogger) [reply config/common.php]: PSR-2 doesn't allow this, if my memory serves me

Comment (razonyang) [reply config/common.php]: @viktorprogger Yes, it doesn't mentioned in PSR-2, it is a new features of php7.0, and there is a proposal called [Extended Coding Style Guide](https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md)

Comment (samdark) [reply config/common.php]: Yes, that is OK.

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**